doc/man3/RAND_add.pod

```diff
@@ -54,6 +57,15 @@ should consider using L<RAND_load_file(3)> instead.
 
 RAND_seed() is equivalent to RAND_add() with B<randomness> set to B<num>.
 
+RAND_keep_random_devices_open() is used to control file descriptor
```

Comment:
descriptors?

Reply:
It's "file descriptor use", but maybe @paulidale should break the line after "use", otherwise more readers will stumble over it.

Reply:
Or "file descriptor usage"?

Reply:
Oh..right. My mistake. Actually the line break is probably not that big a deal because it won't necessarily be there when this gets rendered as a man page.